packit/upstream.py

```diff
@@ -28,15 +28,9 @@
 
 import git
 from packaging import version
+from packit.utils import is_a_git_ref, run_command
 from rebasehelper.exceptions import RebaseHelperError
 
-try:
-    # ogr < 0.5
-    from ogr import GithubService
-except ImportError:
-    # ogr >= 0.5
-    from ogr.services.github import GithubService
```

Comment:
We need new ogr for loading authentication so it is not needed anymore.

Could I mark the `ogr>=0.7.0` requirement in the setup.cfg, or in the specfile?

Reply:
`setup.cfg` should be fine I guess, the [automatic Python dependency generator](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_automatically_generated_dependencies) should take care of it for rpm

Reply:
Done, there was already the requirement for `0.6.0`.